portfolio/src/main/java/com/google/sps/data/UserAuthentication.java

```diff
@@ -5,10 +5,12 @@
   private Boolean isLoggedIn;
```

Comment:
These variables should be final (can only be assigned once) unless there is a need for them to change after instantiation